sphinx/writers/latex.py

```diff
@@ -446,6 +446,18 @@ def __init__(self, document, builder):
             self.elements['multilingual'] = '%s\n\\setmainlanguage{%s}' % \
                 (self.elements['polyglossia'], self.babel.get_language())
 
+        # detect Chinese language, disable babel/polyglossia
+        # give warnings when no common chinese package found in configuration
+        if builder.config.language:
+            if builder.config.language.split('_')[0] == 'zh':
+                self.elements['babel'] = ''
+                self.elements['polyglossia'] = ''
+                if 'ctex' not in self.elements['usepackages'].lower():
```

Comment:
Yes it is intended. ctex is the usually choice in most cases.
Chinese user would always get an warning by babel because babel does not support Chinese.
So I think it is usual that common user would use ctex and advanced user would understand what this warning means.
If your opinion is no warning in proper working cases, I'll think about it more deep